Makefile

```diff
@@ -2,6 +2,10 @@ SHELL = /bin/bash
 COMMIT = $(shell git rev-parse --short HEAD)$(shell [[ $$(git status --porcelain) = "" ]] || echo -dirty)
 ARO_IMAGE ?= ${RP_IMAGE_ACR}.azurecr.io/aro:$(COMMIT)
 
+# disable CGO so we're not dependent on system libraries
+# TODO remove this when we move to only multistage builds
+export CGO_ENABLED=0
```

Comment:
I think this break mirroring code which is where we need system libs.. Not sure this works here at all at this point